click/decorators.py

```diff
@@ -115,13 +116,43 @@ def decorator(f):
     return decorator
 
 
-def group(name=None, **attrs):
+def group(name=None, plugins=None, **attrs):
```

Comment:
Good point - it would probably make sense to put this in `click.core.Group()`.  I wasn't really thinking about how the decorators are technically just provided for convenience.